rcsbsearchapi/search.py

```diff
@@ -1474,8 +1474,8 @@ def _single_query(self, start=0) -> Optional[Dict]:
         logging.debug(
             "Querying %s for results %s-%s", self.url, start, start + self.rows - 1
         )
-        response = requests.get(
-            self.url, {"json": json.dumps(params, separators=(",", ":"))}, timeout=None
+        response = requests.post(
+            self.url, data=params, timeout=None
```

Comment:
Just to confirm, after making this change, do the tests yield the exact same results as when it was set to `.get()`?